h2o-genmodel/src/main/java/hex/genmodel/algos/tree/SharedTreeMojoModel.java

```diff
@@ -30,15 +30,15 @@
      * multiclass response.
      */
     protected int _ntree_groups;
-    protected int _ntrees_per_group;
+    public int _ntrees_per_group;
```

Comment:
@wendycwong can you please make these protected again? they are not used outside of this class

Reply:
Done.  I forgot.  Sorry.